data/distro.yml

```diff
@@ -49,6 +49,32 @@
         </code></pre>
       </p>
     </ol>
+    
+- name: "Manjaro"
+  logo: "manjaro.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Enable Flatpak through the Software Manager</h2>
+        <p>To enable Flatpak through the Software Manager, you must be running <strong>Manjaro 20 or higher</strong></p>
```

Comment:
I would rewrite this to say for example: `Flatpak is installed by default on Manjaro 20 or higher. To enable its support, navigate to the <strong>Software manager</strong>(Add/Remove Programs) ...`

Reply:
Yeah that makes sense, I'll make the changes